common/src/main/java/oracle/kubernetes/common/utils/SchemaConversionUtils.java

```diff
@@ -116,7 +113,6 @@ public Resources convertDomainSchema(Map<String, Object> domain, ResourceLookup
     if (spec == null) {
       return new Resources(domain, Collections.emptyList());
     }
-    LOGGER.fine("Converting domain " + domain + " to " + targetAPIVersion + " apiVersion.");
```

Comment:
I assume we don't need this logging when running the standalone command line utility to upgrade the domain from `v8` to `v9`. The REST API won't be invoked when running the standalone command line utility.